ignition/configuring-the-network.md

```diff
@@ -0,0 +1,68 @@
+# Configuring the Network #
+
+Configuring networkd with Ignition is a very straightforward task. Because
+Ignition runs before networkd starts, configuration is just a matter of writing
+the desired config to disk. The Ignition config has a specific section
+dedicated to this.
+
+## Static Networking ##
+
+In this example, the network interface with the name "eth0" will be given the
+IP address 10.0.1.7. A typical interface will need more configuration and can
+use all of the options of a [network unit][network].
+
+```json
+{
+	"networkd": {
+		"units": [
+			{
+				"name": "00-eth0.network",
+				"contents": "[Match]\nName=eth0\n\n[Network]\nAddress=10.0.1.7"
```

Comment:
I wish there was an easier way to read these things. I know you're reproduce it below, but when users are auditing their configs it is exceptionally hard to read.